### DISCUSSION TITLE
Making a VM from a custom dd disk image

### DISCUSSION THREAD

**Discussion by selroc** [cloud-hypervisor/cloud-hypervisor#5605] [Q&A] [answered]
I'd like to make a custom VM using dd to save a disk and then pass it to cloud-hypervisor.
Are there any particular steps I must follow to achieve this?

**Comment by liuw** [comment, marked as answer]
If you use direct kernel boot, I don't think there is anything special required.

If you want to do EFI boot with either OVMF and hypervisor-fw, make sure the disk image (or the source disk your copy from) is EFI capable.

**Reply by selroc** [reply]
Thank you.
